GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+# Fn Project Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Fn Project community. Common terminology used in this governance document are listed below:
+  * Team members: members of the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * Maintainers: Maintainers lead an individual project or parts thereof.
+  * Projects: A single repository in the [Fn Project GitHub organization](https://github.com/fnproject).
+  * The Fn Project: The sum of all activities performed under this governance, concerning one or more repositories or the community. 
+
+## Values
+The Fn Project exists to serve the CNCF ecosystem. Thus developers and community are expected to follow the values defined in the [CNCF Charter](https://www.cncf.io/about/charter/), including the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Furthermore, the Fn Project community strives for kindness, giving feedback effectively, and building a welcoming environment. The Fn Project developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Projects
+Each project must have a `MAINTAINERS.md` file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced in the [Release category in forums.fnproject.io](https://forums.fnproject.io/c/release) . Any new projects should be first proposed in the [Fn Project forums](https://forums.fnproject.io/) following the voting procedures listed below. When a project is no longer relevant it should be moved to the fnproject-junkyard GitHub organization.
+
+## Decision 
+### Team members
+Team member status may be given to those who have made ongoing contributions to the Fn Project for at least 3 months. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support will also be taken into account.
+
+New members may be proposed by any existing member by posting to the Fn Project forums. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
```

Comment:
I wonder if there is a possible case where someone feels strongly that a person should be blocked from joining the team but does not want to discuss their reasons publicly - I'm imagining some victimisation or harassment - should a maintainer be allowed to veto?  A team member can do significant harm and is difficult to remove after the fact.

Reply:
this is a good point. this paragraph also does not stipulate whether members are voting or maintainers are voting. probably, if any maintainer doesn't feel comfortable with a person they should be trusted [due to their status] to be able to reasonably state (in private, perhaps) that reason pursuant to e.g. the CoC we have in place -- if not, maybe the maintainer is the issue. overall, this doesn't concern me too much in practice but we should have a solid process no less.